src/hooks/useFetchData.ts

```
@@ -0,0 +1,34 @@
+import { useCallback } from 'react';
+
+interface useFetchDataProps {
+  dataName: string;
+}
+
+interface FetchDataProps<T> {
+  apiCall: () => Promise<T | void>;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)